src/Manager.sol

```diff
@@ -267,18 +267,12 @@ abstract contract Manager is
         view
         returns (uint256 capacity)
     {
-        capacity = rateLimitParams.currentCapacity;
-        if (capacity == rateLimitParams.limit) {
-            return capacity;
-        } else if (rateLimitParams.lastTxTimestamp + rateLimitDuration <= block.timestamp) {
-            capacity = rateLimitParams.limit;
-        } else if (rateLimitParams.lastTxTimestamp + rateLimitDuration > block.timestamp) {
-            uint256 timePassed = block.timestamp - rateLimitParams.lastTxTimestamp;
-            uint256 calculatedCapacity = capacity + (timePassed * rateLimitParams.ratePerSecond);
-            capacity = calculatedCapacity > rateLimitParams.limit
-                ? rateLimitParams.limit
-                : calculatedCapacity;
-        }
+        uint256 timePassed = block.timestamp - rateLimitParams.lastTxTimestamp;
+        uint256 calculatedCapacity =
+            rateLimitParams.currentCapacity + (timePassed * rateLimitParams.ratePerSecond);
+
+        return
+            calculatedCapacity > rateLimitParams.limit ? rateLimitParams.limit : calculatedCapacity;
```

Comment:
might be good to abstract this out into a `min` or `max` function to make it more consistent with the other implementations. also makes the clamping explicit.

Reply:
also imo a bit more readable if its `calculatedCapacity < rateLimitParams.limit ? calculatedCapacity: rateLimitParams.limit`